chroma_core/models/stratagem.py

```diff
@@ -687,7 +687,7 @@ def get_steps(self):
                 {
                     "client_args": (
                         client_host.fqdn,
-                        client_mount.mountpoint,
+                        client_mount.mountpoints[0],
```

Comment:
Are we guaranteed that mountpoints isn't empty?

Reply:
Yes, the invariant is the same as before, now it's just a list.